.github/workflows/tests.yml

```diff
@@ -134,9 +134,11 @@ jobs:
     strategy:
       matrix:
         python:
+          - 3.10
```

Comment:
Using `3.10` does not work, because the GitHub Action worker believes this is a float and uses `3.1` (this is why the CI tests failed). This value needs to be quoted, which is what I did in PR https://github.com/chipsec/chipsec/pull/1533.
Now that this other PR is merged, this PR can be updated to only add Python 3.6.

Reply:
Yeah, am going to update this. Thanks for pointing this out though!